### PR TITLE
Action button edits

### PR DESCRIPTION
**Pull request by Erin-Cecele**

### CODE REVIEW

File: static/css/custom.css [hunk 1]
```diff
@@ -82,6 +82,24 @@
   color: white !important;
   background-color: var(--accent-color-dark);
 }
+/* Button-like links used for calls to action centered on page
```
Comment (geealbers) [inline static/css/custom.css:85]: Cool. Definitely works, but is there a reason we'd need to have one thing that's an action-button and a second thing that's an action-button-center? Aren't they always centered? If so, this is quite a bit of duplicate code.

Comment (Erin-Cecele) [reply static/css/custom.css:85]: For me, the centered buttons look really weird on the "Join Us". 

Here is the current layout: 

Screen Shot 2020-09-10 at 3 06 50 PM

Here is the page with everything centered:

Screen Shot 2020-09-10 at 3 11 20 PM

If you think the latter looks ok though then I am all for simplifying.

Comment (geealbers) [reply static/css/custom.css:85]: I see what you're saying. though the left button and small left image look a little weird to me too. too much white space to the right there, followed by a narrow band of text. maybe we need an alternate solution there. I can play with it a bit. Is there anywhere else (so far) where a centered button doesn't work?

**Comment by Erin-Cecele** [comment]
It's fine everywhere else. What do you think of having the action button text bold everywhere too? The only place it really looked weird/too loud to me was on homepage. 

And, yes, I'd love your input on formatting the Join Us page. I'm happy with how it's coming along but it does need help. Thank you!

**Pull request merged**